### PR TITLE
Fix disable tests targets

### PR DESCRIPTION
**Pull request by azubieta**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ project(AppImageKit)
 include(ExternalProject)
 
 # enable testing globally
+option(BUILD_TESTING "Build the testing tree" ON)
```
[the remark this answers is not in the document]
Comment (azubieta) [reply CMakeLists.txt]: It seems that we are good without using the `option` nor the `set` commands. Reverting that commit.

**Comment by TheAssassin** [comment]
Tested it, the option doesn't have to be defined, but GTest was still built so I wrapped the GTest dependency stuff in an `if` block as well.

**Comment by TheAssassin** [comment]
Thanks!

**Comment by probonopd** [comment]
Thank you

**Pull request merged**